JS/andOr.js

```diff
@@ -0,0 +1,7 @@
+let isFun = true;
+let isLearning = 0;
+
+console.log(isFun && "Learning is fun!");
+console.log(isFun || "Learning is not fun!");
```

Comment:
Did you meant to use isLearning here, instead of isFun? (isLearning variable is not used in the entire code, so was doubtful!)

Reply:
Yeah changing it : )..